qgroundcontrol.qrc

```diff
@@ -296,18 +296,20 @@
         <file alias="CameraGalleryButton.qml">src/QmlControls/CameraGalleryButton.qml</file>
         <file alias="QGroundControl/Controls/Grid.qml">src/QmlControls/Grid.qml</file>
         <file alias="QGroundControl/Controls/CenterPoint.qml">src/QmlControls/CenterPoint.qml</file>
-        <file alias="CommonTopMenu.qml">src/QmlControls/CommonTopMenu.qml</file>
+        <file alias="QGroundControl/Controls/CommonTopMenu.qml">src/QmlControls/CommonTopMenu.qml</file>
         <file alias="QGroundControl/Controls/CameraSettings.qml">src/QmlControls/CameraSettings.qml</file>
         <file alias="QGroundControl/Controls/AntiFlicker.qml">src/QmlControls/AntiFlicker.qml</file>
-		    <file alias="QGroundControl/Controls/CameraBottomMenu.qml">src/QmlControls/CameraBottomMenu.qml</file>
-        <file alias="LearningCenterDialog.qml">src/QmlControls/LearningCenterDialog.qml</file>
-        <file alias="PreflightSecurityCheck.qml">src/QmlControls/PrelightSecurityCheck.qml</file>
-        <file alias="PreflightSecurityCheckV1.qml">src/QmlControls/PrelightSecurityCheckV1.qml</file>
-        <file alias="RemoteControlInstruction.qml">src/QmlControls/RemoteControlInstruction.qml</file>
-        <file alias="SdCardDialog.qml">src/QmlControls/SdCardDialog.qml</file>
-        <file alias="MissionHistoryItem.qml">src/QmlControls/MissionHistoryItem.qml</file>
+		<file alias="QGroundControl/Controls/CameraBottomMenu.qml">src/QmlControls/CameraBottomMenu.qml</file>
+        <file alias="QGroundControl/Controls/LearningCenterDialog.qml">src/QmlControls/LearningCenterDialog.qml</file>
+        <file alias="QGroundControl/Controls/PreflightSecurityCheck.qml">src/QmlControls/PrelightSecurityCheck.qml</file>
+        <file alias="QGroundControl/Controls/PreflightSecurityCheckV1.qml">src/QmlControls/PrelightSecurityCheckV1.qml</file>
+        <file alias="QGroundControl/Controls/RemoteControlInstruction.qml">src/QmlControls/RemoteControlInstruction.qml</file>
+        <file alias="QGroundControl/Controls/SdCardDialog.qml">src/QmlControls/SdCardDialog.qml</file>
+        <file alias="QGroundControl/Controls/MissionHistoryItem.qml">src/QmlControls/MissionHistoryItem.qml</file>
```

Comment:
What is the reason behind this changing path?

Reply:
The files in other QmlControls folders are named the following, so they are just set the same.